Packs/Jizo_M/Integrations/JizoM/JizoM.yml

```diff
@@ -0,0 +1,434 @@
+      description: The number of displayed alerts.
+      type: Number
+    - contextPath: JizoM.Protocols.alerts_files.data
+      description: The details of alerts.
+      type: String
+    - contextPath: JizoM.Protocols.alerts_files.total
+      description: The total number of alerts.
+      type: Number
+    - contextPath: JizoM.Protocols.alerts_usecase.count
+      description: The number of displayed alerts.
+      type: Number
+    - contextPath: JizoM.Protocols.alerts_usecase.data
+      description: The details of alerts.
+      type: String
+    - contextPath: JizoM.Protocols.alerts_usecase.total
+      description: The total number of alerts.
+      type: Number
+    description: Get jizo protocols.
+  - name: jizo-m-peers-get
+    arguments:
+    - name: ip_src
+      description: Ipv4 or Ipv6 of the source.
+      type: textArea
+    - name: ip_dest
+      description: Ipv4 or Ipv6 of the destination.
+      type: textArea
+    - name: datetime_from
+      description: Get the alerts that were occurred from this date. The default value is 7 days ago.
+      type: textArea
+    - name: datetime_to
+      description: Get the alerts that were occurred up to this date. The default is now.
+      type: textArea
+    - name: probe_name
+      description: The name of the jizo probe.
+      type: textArea
+    - name: page
+      description: A page number for pagination.
+      type: keyValue
+    - name: limit
+      description: The maximum number of samples to display per alert.
+      type: keyValue
+    outputs:
+    - contextPath: JizoM.Peers.alerts_flows.count
+      description: The number of displayed alerts.
+      type: Number
+    - contextPath: JizoM.Peers.alerts_flows.data
+      description: The details of alerts.
+      type: String
+    - contextPath: JizoM.Peers.alerts_flows.total
+      description: The total number of alerts.
+      type: Number
+    - contextPath: JizoM.Peers.alerts_files.count
+      description: The number of displayed alerts.
+      type: Number
+    - contextPath: JizoM.Peers.alerts_files.data
+      description: The details of alerts.
+      type: String
+    - contextPath: JizoM.Peers.alerts_files.total
+      description: The total number of alerts.
+      type: Number
+    - contextPath: JizoM.Peers.alerts_usecase.count
+      description: The number of displayed alerts.
+      type: Number
+    - contextPath: JizoM.Peers.alerts_usecase.data
+      description: The details of alerts.
+      type: String
+    - contextPath: JizoM.Peers.alerts_usecase.total
+      description: The total number of alerts.
+      type: Number
+    description: Get jizo peers.
+  - name: jizo-m-query-records-get
+    arguments:
+    - name: ip_src
+      description: Ipv4 or Ipv6 of the source.
+      type: textArea
+    - name: ip_dest
+      description: Ipv4 or Ipv6 of the destination.
+      type: textArea
+    - name: proto
+      description: The protocol (e.g., TCP, UDP).
+      type: textArea
```

Comment:
If there is a close list of application protocols it is recommended to use a pre-defined list the customer can choose from.
As in this example from the Hello World pack :

```
name: severity
isArray: false
description: Filter by alert severity. Comma-separated value (Low,Medium,High,Critical).
required: true
secret: false
default: false
predefined:
- Low
- Medium
- High
- Critical
auto: PREDEFINED
```

Reply:
done